bin/kmd

```diff
@@ -97,17 +96,28 @@ def main(args):
                                    os.environ.get('PATH', '')])
 
     print('Fixup #! in pip-provided packages')
+    if 'python' in pkgs.keys():
+        # Currently we need to keep pip to 18.1 due to a bug in pip 19<
+        # https://github.com/pypa/pip/issues/6327
+        print(komodo.shell(['{}/bin/pip'.format(install_root),
+                            'install --upgrade pip==18.1',
+                            '--prefix', os.path.join(args.prefix, args.release, 'root'),
+                            '--force-reinstall']))
+
     for pkg, ver in pkgs.items():
         if repo[pkg][ver]['make'] != 'pip': continue
 
-        print(komodo.shell(['{}/bin/pip'.format(install_root),
-               'install {}=={}'.format(pkg, ver),
-               '--prefix', os.path.join(args.prefix, args.release, 'root'),
-               '--force-reinstall',
-               '--no-index',
-               '--find-links {}'.format(args.cache),
-               repo[pkg][ver].get('makeopts')
-              ], sudo = args.sudo))
+        stripped_ver = ver.split("+")[0]
+        shell_input = [ '{}/bin/pip'.format(install_root),
+                        'install {}=={}'.format(pkg, stripped_ver),
+                        '--prefix', os.path.join(args.prefix, args.release, 'root'),
+                        '--force-reinstall',
+                        '--no-index',
+                        '--no-deps',
+                        '--find-links {}'.format(args.cache),
+                        repo[pkg][ver].get('makeopts')]
+
+        print(komodo.shell(shell_input, sudo = args.sudo))
```

Comment:
`sudo=args.sudo`

Reply:
I see that my comments regarding white spacing and named arguments are not in line with the rest of the code and that the rest of the code is not in line with PEP-8. Do as you like ;)